osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs

```diff
@@ -280,8 +280,8 @@ private void handleTimingPoint(string line)
             {
                 string[] split = line.Split(',');
 
-                double time = getOffsetTime(double.Parse(split[0].Trim(), NumberFormatInfo.InvariantInfo));
-                double beatLength = double.Parse(split[1].Trim(), NumberFormatInfo.InvariantInfo);
+                double time = getOffsetTime(Parsing.ParseDouble(split[0].Trim()));
+                double beatLength = Parsing.ParseDouble(split[1].Trim());
```

Comment:
Why is everything else below not parsed in the same way?

Reply:
Can do; was just focused on fixing the values that would break gameplay initially.